pom.xml

```diff
@@ -1,12 +1,97 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.example</groupId>
+    <groupId>com.epam.training.center.qa</groupId>
     <artifactId>AutoTestProj</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <properties>
+        <testng.version>7.0.0</testng.version>
+    </properties>
+
+
+    <profiles>
+        <profile>
+            <id>SumAndSub</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <suite.file>src/test/resources/homework01/SumAndSub.xml</suite.file>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>MultAndDiv</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <suite.file>${project.basedir}/src/test/resources/homework01/MultAndDiv.xml</suite.file>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>allTest</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <suite.file>${project.basedir}/src/test/resources/homework01/allTtest.xml</suite.file>
+            </properties>
+        </profile>
+    </profiles>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>${testng.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.epam.tat.module4</groupId>
+            <artifactId>calculator</artifactId>
+            <version>1.0</version>
+            <scope>system</scope>
+            <systemPath>
+                ${project.basedir}/src/test/resources/Calculator/calculator-1.0.jar
```

Comment:
:heavy_exclamation_mark: :heavy_exclamation_mark: :heavy_exclamation_mark: Look, interesting thing:

1) you write in .gitignore:
![image](https://user-images.githubusercontent.com/44112289/101688130-9d0a8480-3a7c-11eb-9bbf-a64da678fe0b.png)

2) you write in pom.xml:
![image](https://user-images.githubusercontent.com/44112289/101688172-af84be00-3a7c-11eb-9dbb-442f3d0c9d11.png)


It's happen because you exclude all .jar files from files to be commited in git